dawn/src/dawn/SIR/proto/SIR/statements.proto

```diff
@@ -461,7 +478,8 @@ message VarAccessExpr {
   Expr index = 2;         // Is it an array access (i.e var[2])?
   bool is_external = 3;   // Is this an access to a external variable (e.g a global)?
   SourceLocation loc = 4; // Source location
-  int32 ID = 5;           // ID of the Expr
+  AccessExprData data = 5; // Access data
```

Comment:
in protobuf3 `data` is already optional ... since there won't be any SIR data for expressions, do you think it is still useful to put `OneOf` between `AccessExprData` and empty?

Reply:
I agre maybe it is only useful if we had SIR data. Do you think it still makes sense to keep data in a separate message?

Reply:
To me it's still (kind of) useful to keep a logical separation between what is proper AST information and what is a "decoration" added by components processing it